docs/site/Validation-REST-layer.md

```diff
@@ -0,0 +1,163 @@
+---
+lang: en
+title: 'Validation in REST Layer'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Validation-REST-layer.html
+---
+
+At the REST layer, there are 2 types of validation:
+
+1. [Type validation](#type-validation)
+2. [Validation against OpenAPI schema specification](#validation-against-openapi-schema-specification)
+
+## Type Validation
+
+The type validation in the REST layer comes out of the box in LoopBack.
+
+> Validations are applied on the parameters and the request body data. They also
```

Comment:
> Validations are applied on the parameters and the request body data. They also use OpenAPI specification as the reference to infer the validation rules.

Validation is applied on the parameters and the request body data. It also uses the OpenAPI specification as a reference to infer the validation rules.